### PR TITLE
Update coding style and contribution docs

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**

### CODE REVIEW

File: docs/contributing-analyzer.md [hunk 1]
```diff
@@ -15,16 +15,17 @@ In general, it is best to run commands from the Visual Studio Developer Command
 1. [Visual Studio 2019](https://visualstudio.microsoft.com/vs/)
 1. Ensure to install Workloads:
```
Comment (pavel-mikula-sonarsource) [inline docs/contributing-analyzer.md:16]: Adjusted to VS 2019 installer names

File: docs/contributing-analyzer.md [hunk 2]
```diff
@@ -15,16 +15,17 @@ In general, it is best to run commands from the Visual Studio Developer Command
...
-    - .NET desktop development (C#, VB and F# with .NET Core and .NET Framework)
+    - .NET desktop development
     - Visual Studio extension development
+    - .NET Core cross-platform development
 1. Ensure to install Individual components:
     - .NET Core 3.1 SDK
     - .NET Framework 4.8 SDK
     - .NET Framework 4.8 Targeting pack
     - .NET 3.5 SDK (SP1)
+    - .NET Compiler Platform SDK
```
Comment (pavel-mikula-sonarsource) [inline docs/contributing-analyzer.md:26]: To get syntax visualizer

File: docs/contributing-analyzer.md [hunk 3]
```diff
@@ -15,16 +15,17 @@ In general, it is best to run commands from the Visual Studio Developer Command
...
 1. The following environment variables must be set:
     - **JAVA_HOME**
-    - **MAVEN_HOME** / **M2_HOME**
```
Comment (pavel-mikula-sonarsource) [inline docs/contributing-analyzer.md:27]: Unused. I don't have them on my machine (never had) and I haven't found them anywhere in the project.

**Review by andrei-epure-sonarsource — APPROVED** [review]
LGTM

File: docs/contributing-analyzer.md (hunk 3 above)
Comment (andrei-epure-sonarsource) [reply docs/contributing-analyzer.md:27]: I probably added everything I had locally, and maybe this was needed in other projects

**Pull request merged**